frontend/src/concepts/areas/__tests__/utils.spec.ts

```diff
@@ -190,46 +190,8 @@ describe('isAreaAvailable', () => {
     });
 
     describe('requiredCapabilities', () => {
-      it('should enable area if both capabilities are enabled', () => {
-        // Make sure this test is valid
-        expect(SupportedAreasStateMap[SupportedArea.K_SERVE_AUTH].requiredCapabilities).toEqual([
-          StackCapability.SERVICE_MESH,
-          StackCapability.SERVICE_MESH_AUTHZ,
-        ]);
-
-        // Test both reliant areas
-        const isAvailableKserveAuth = isAreaAvailable(
-          SupportedArea.K_SERVE_AUTH,
-          mockDashboardConfig({ disableKServeAuth: false }).spec,
-          mockDscStatus({
-            installedComponents: {
-              [StackComponent.K_SERVE]: true,
-            },
-          }),
-          mockDsciStatus({
-            requiredCapabilities: [
-              StackCapability.SERVICE_MESH,
-              StackCapability.SERVICE_MESH_AUTHZ,
-            ],
-          }),
-        );
-
-        expect(isAvailableKserveAuth.status).toBe(true);
-        expect(isAvailableKserveAuth.featureFlags).toEqual({
-          disableKServeAuth: 'on',
-        });
-        expect(isAvailableKserveAuth.requiredCapabilities).toEqual({
-          [StackCapability.SERVICE_MESH]: true,
-          [StackCapability.SERVICE_MESH_AUTHZ]: true,
-        });
-      });
-
       it('should enable area if one capability is missing', () => {
         // Make sure this test is valid
-        expect(SupportedAreasStateMap[SupportedArea.K_SERVE_AUTH].requiredCapabilities).toEqual([
```

Comment:
Same for the rest of this file - we don't need to touch any of the tests related to kserve, we should revert all the changes in this file